.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "microsoft.dotnet-httprepl": {
+      "version": "7.0.0",
+      "commands": [
+        "httprepl"
```

Comment:
You probably don't want this file checked in (or at least not with httprepl in it)